README.md

```diff
@@ -2,7 +2,18 @@
 
 __`lava-dl`__ is a library of deep learning tools within Lava that support  offline training, online training and inference methods for various Deep Event-Based Networks.
 
-The library presently consists of 
+There are two main strategies for training Deep Event-Based Networks: _direct training_ and _ANN to SNN converison_. 
+
+Directly training the network utilizes the information of precise events in time. It is very accurate and results in efficient networks. However, the training these networks take a lot of time and resources.
+
+On the other hand, ANN to SNN conversion is especially suitable for rate coded SNNs where we can leverage the fast training of ANN. These converted SNNs, however, require increased latency compared to directly trained SNNs.
+
+Lava-DL provides improved [SLAYER](https://github.com/bamsumit/slayerPytorch) for direct training of deep event based networks and a new ANN-SNN accelerated training approach called [Bootstrap:TODO](link_here) to mitigate low latency issue of conventional ANN-SNN methods for training Deep Event-Based Networks.
+
+The Lava-DL training api's are independent of core Lava api since there is no native support for backpropagation in Lava processes. However, these training libraries support platform independent hdf5 network description protocol. Lava-DL also includes Network exchange library (coming soon...) that automatically generates the trained network using Lava processes and enables inference of the deep networks through lava on various backends.
```

Comment:
-> API, not api;  but butter to say:
The lava-dl library is independent of the core lava library since Lava Processes cannot be trained directly at this point. Instead, lava-dl is first used to train an independent SLAYER/PyTorch?-model which can then be converted to a network of Lava processes using the netx library.

Reply:
Reworded to:
The lava-dl training libraries are independent of the core lava library since Lava Processes cannot be trained directly at this point. Instead, lava-dl is first used to train the model which can then be converted to a network of Lava processes using the netx library using platform independent hdf5 network description.